general.ruleset

```diff
@@ -429,4 +430,4 @@
     <Rule Id="SX1101" Action="Warning" />
     <Rule Id="SX1309S" Action="Warning" />
   </Rules>
-</RuleSet>
+</RuleSet>
```

Comment:
Please don't remove empty lines at the end of files.

Reply:
The VS graphical editor for rulesets removes it. Just as we changed the file name casing due to it constantly doing that, I wouldn't keep re-adding this just for the sake of it.

Reply:
I see. Makes sense, although it also makes me sad.

Reply:
Me too.